### PR TITLE
Correct button copy 

### PR DESCRIPTION
**Pull request by jprcoliveira**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by akeller — COMMENTED** [review]
Should this be in `/docs/` as well? So it shows up for 8.4 or the preview versions?

**Comment by jprcoliveira** [comment]
> Should this be in `/docs/` as well? So it shows up for 8.4 or the preview version?

I was working on it. My browser just crashed. Now should be good.

**Comment by jprcoliveira** [comment]
Should be good to go now @akeller

**Review by akeller — APPROVED** [review]

File: docs/components/console/manage-plan/retrieve-invoices-or-update-billing-info.md
Comment (christinaausley) [inline docs/components/console/manage-plan/retrieve-invoices-or-update-billing-info.md:1]: Should line 19 be adjusted as well?

File: versioned_docs/version-8.3/components/console/manage-plan/retrieve-invoices-or-update-billing-info.md
Comment (christinaausley) [inline versioned_docs/version-8.3/components/console/manage-plan/retrieve-invoices-or-update-billing-info.md:1]: Same here for line 19 👍

**Review by christinaausley — CHANGES_REQUESTED** [review]
Two comments 👍

**Comment by christinaausley** [comment]
@jprcoliveira I adjusted line 19 but let me know if this was a mistake and can adjust in a subsequent PR 👍

**Review by christinaausley — APPROVED** [review]

**Pull request merged**